wasm/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ target_include_directories(bergamot-translator-worker
 target_compile_definitions(bergamot-translator-worker PRIVATE WASM_BINDINGS)
 target_compile_options(bergamot-translator-worker PRIVATE ${WASM_COMPILE_FLAGS})
 
-set(LINKER_FLAGS "--bind -s ASSERTIONS=0 -s DISABLE_EXCEPTION_CATCHING=1 -s FORCE_FILESYSTEM=1 -s ALLOW_MEMORY_GROWTH=1 -s NO_DYNAMIC_EXECUTION=1")
+set(LINKER_FLAGS "-g2 --bind -s ASSERTIONS=0 -s DISABLE_EXCEPTION_CATCHING=1 -s FORCE_FILESYSTEM=1 -s ALLOW_MEMORY_GROWTH=1 -s NO_DYNAMIC_EXECUTION=1 -s EXPORTED_RUNTIME_METHODS=[addOnPreMain]")
```

Comment:
> It could help to know what does -g2 does?

This flag adds debugging information to the binary. It is a very commonly used flag (exists for clang, gcc compiler as well), so I didn't add anything to document.

> I'm assuming the other edit follows [mozilla-extensions/bergamot-browser-extension#122 (comment)](https://github.com/mozilla-extensions/bergamot-browser-extension/pull/122#discussion_r629514844).

Correct 👍